internal/worker/state/manifold_test.go

```diff
@@ -1,275 +0,0 @@
-// Copyright 2016 Canonical Ltd.
-// Licensed under the AGPLv3, see LICENCE file for details.
-
-package state_test
```

Comment:
These are fine to nuke. Add a separate PR for it and I'll approve.

Reply:
https://github.com/juju/juju/pull/19223